pkg/rbd/controllerserver.go

```diff
@@ -115,6 +115,15 @@ func parseVolCreateRequest(req *csi.CreateVolumeRequest) (*rbdVolume, error) {
 	return rbdVol, nil
 }
 
+func storeVolumeMetadata(vol *rbdVolume, cp util.CachePersister) error {
+	if err := cp.Create(vol.VolID, vol); err != nil {
```

Comment:
any special handling if the err is `already exist`?

Reply:
Already taken care of :)

https://github.com/ceph/ceph-csi/blob/8223ae325b0bd9ab232a3f405737bd8146ffac06/pkg/util/k8scmcache.go#L145-L152